calc/async_calc_graph.go

```diff
@@ -149,6 +150,9 @@ func (acg *AsyncCalcGraph) loop() {
 					typeName := reflect.TypeOf(upd.Key).Name()
 					count := countUpdatesProcessed.WithLabelValues(typeName)
 					count.Inc()
+					if time.Since(acg.lastReport) > healthInterval {
```

Comment:
Is the elapsed time check really useful here?  It would be simpler to call `acg.reportHealth` unconditionally, so should we just do that?

Reply:
Thinking was that it might cause a lot of log spam from the health aggregator?

Reply:
I think the aggregator report-time logging is all debug, and it might be better if needed to handle any log rate limiting inside the aggregator?  I think I'd prefer to err initially on the side of over-reporting from Felix.

Reply:
Benchmarked the report function at 500ns; that's probably OK.  Removing the logCtx from the report function knocks it down to 90ns in case we need to optimise that in future.